packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx

```diff
@@ -59,55 +140,6 @@ test("should emit events", async ({ mount, makeAxeBuilder }) => {
   });
 });
 
-test("should show required marker", async ({ mount }) => {
-  // ARRANGE
-  const component = await mount(<OnyxInput label="Label" style="width: 12rem;" required />);
-
-  // ASSERT
-  await expect(component).toHaveScreenshot("required.png");
-});
-
-test("should show optional marker", async ({ mount }) => {
-  // ARRANGE
-  const component = await mount(
-    <OnyxInput
-      label="Very long label that should be truncated"
```

Comment:
we have no truncation example anymore, or did I miss it?

Reply:
maybe combine that with the long label with required + optional
and you could also pass in a very long placeholder / filled text.

Reply:
Good catch, I added it and also for the invalid state